workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/GameObjectCreation/GameObjectInitializationSystem.cs

```diff
@@ -38,51 +32,40 @@ private struct RemovedEntitiesData
 
         private WorkerSystem worker;
         private ViewCommandBuffer viewCommandBuffer;
-        private EntityGameObjectCreator entityGameObjectCreator;
         private EntityGameObjectLinker entityGameObjectLinker;
+        private EntityManager entityManager;
```

Comment:
that should not be needed. Each system already has a reference to the `EntityManager`